### ISSUE TITLE
React.DOM.noscript Warning

### ISSUE THREAD

**Issue by andrewjmead** [reactjs/react-modal#411]
First up, thanks for helping push this library forward!

Just wanted to make you aware of this new warning as I didn't see an open issue about it. Hopefully this will be of some help.

### Summary:

As of [React 15.6](https://github.com/facebook/react/releases/tag/v15.6.0), there is a deprecation warning when accessing React.DOM factory helpers. Here's the warning in full:

```
Warning: Accessing factories like React.DOM.noscript has been deprecated and will be removed in v16.0+. Use the react-dom-factories package instead.  Version 1.0 provides a drop-in replacement.
```

The usage can be seen here: [dist/react-modal.js line 173](https://github.com/reactjs/react-modal/blob/3ada4fba8133db5750decb72cede1992c5d8a40a/dist/react-modal.js#L173)

### Steps to reproduce:

 1. Use the latest React version (15.6.0) with the latest React-Modal version (1.9.4).

**Comment by diasbruno** [comment]
Hi @andrewjmead, there is PR #410 (what you are looking for) and #395. Do mind review both PRs?

**Comment by andrewjmead** [comment]
Yes. This is my fault. I looked though the issue list, but I failed to look though the PR list.

**Comment by diasbruno** [comment]
Awesome. I'll release a new version with #410.

**Comment by diasbruno** [comment]
Thank you so much @andrewjmead! Released `v1.9.6`.

**Comment by andrewjmead** [comment]
Very cool. Thanks for your work. I've been using react-modal for years and would love to get a little more involved. See you around!

**Comment by diasbruno** [comment]
Awesome. Appreciate your help. We are close to get `v2.0.0` merged, I'm trying to get the last PRs reviewed and merged to finally merge #412.
